resources/uploads/attributes/column_mapping.yml

```diff
@@ -3,4 +3,12 @@ type: object
 example:
   name: recipient_name
   address_line1: primary_line
-description: The mapping of column headers in your file to Lob-required fields for the resource created. See our <a href="https://help.lob.com/best-practices/campaign-audience-guide" target="_blank">Campaign Audience Guide</a> for additional details.
+  address_city: city,
+  address_state: state,
+  address_zip: zip_code
+  address_line2: secondary_line
+  company: company
+  address_country: country,
+  metadata: { columns: ["recipient_name"] }
+  gift_code: code
+description: The mapping of column headers in your file to Lob-required fields for the resource created. See our <a href="https://help.lob.com/best-practices/campaign-audience-guide" target="_blank">Campaign Audience Guide</a> for additional details. <br/>Please note that this field will be **deprecated on 31st October, 2022**. <br/>The new fields that will replace this are `requiredAddressColumnMapping`, `optionalAddressColumnMapping`, `metadata`, and `mergeVariableColumnMapping`.
```

Comment:
Maybe change the language to say that the field IS deprecated now and will be removed on 31st October, 2022 (though I suspect we'll need to move that date back at this point?).

Reply:
That's a great point! I'll do that. 

> though I suspect we'll need to move that date back at this point

Yeah. The plan was to give a grace period of around 5 business days. But if it gets merged in today then i will extend the grace period to 2rd November i think,2nd November being the last day that it can be used. And on 3rd Nov i'll update the docs again to remove it, as well start the cleanup stuff on the BE. So effectively the grace period will be a little beyond 2nd November, but in the public documentation i will remove ALL mentions of it on 3rd Nov.